src/site/static/eligibility.liquid

```diff
@@ -2033,8 +2033,44 @@ pageClass: "page-public-assistance"
   }
 
   function reducedFeeIdEligible() {
-    // TODO
-    return  false;
+    // https://www.dmv.ca.gov/portal/driver-licenses-identification-cards/identification-id-cards/
+    // https://www.icarol.info/ResourceView2.aspx?org=2225&agencynum=73919505
+    //
+    // DL 937 order form:
+    // https://www.dmv.ca.gov/portal/file/order-request-reduced-fee-or-no-fee-identification-card-program-dl-932-pdf/
+    //   Burton-Moscone-Bagley Citizens’ Income Security Act for Aged, Blind, and Disabled Californians (WIC §12000); is called WIC Section 12000 and does not have a seperate progam name, 
+    //   so could be part of the broader WIC benefits progam.
+    //
+    // Other benefit programs for Aged, Blind and Disabled are: IHSS, SSI.
+    const isHoused = selectedOneOf('housing-situation', [
+      'housed',
+      'unlisted-stable-place']);
+    
+    const age = getValueOrNull('age');
+    const MAX_REDUCEDFEEID_ELIGIBLE_AGE = 62;  // Years
+
+    const isProgramQualified = or(
+      document.getElementById("existing-wic-me").checked,
```

Comment:
I think this should be removed, as the WIC referenced in the icarol.org and DL 937 documentation is different from the  Women, Infants and Children program.